### PR TITLE
Rlec fixes final

### PR DESCRIPTION
**Pull request by OfirMos**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -590,6 +654,18 @@ workflows:
       - build-linux-debian:
           name: build
           <<: *not-on-integ-branch
+      - coverage:
+          #@@<<: *on-any-branch
+          <<: *never
```
Comment (rafie) [inline .circleci/config.yml:659]: Please uncomment `on-any-branch` and remove `never`.

File: .circleci/config.yml [hunk 2]
```diff
@@ -590,6 +654,18 @@ workflows:
...
+      - build-memcheck:
+          #@@<<: *on-any-branch
+          <<: *never
```
Comment (rafie) [inline .circleci/config.yml:662]: Please uncomment `on-any-branch` and remove `never`.

**Comment by rafie** [comment]
Please note comments on `config.yml` (some minor fixes).

File: tests/flow/test_rdb_load.py [hunk 3]
```diff
@@ -31,6 +31,8 @@ def _test_data(self):
         self.env.assertEqual(result[1], expected)
     
     def test_rdb_load(self):
+        # DEBUG command on redis cluster is excluded
+        self.env.skipOnCluster()
         aux = self.conn.execute_command("GRAPH.DEBUG", "AUX", "START")
```
Comment (AviAvni) [inline tests/flow/test_rdb_load.py:36]: use restoremodaux

File: tests/flow/test_replication.py [hunk 4]
```diff
@@ -14,10 +14,11 @@ def __init__(self):
         self.env = Env(decodeResponses=True, env='oss', useSlaves=True)
 
         # skip test if we're running under Valgrind
-        if self.env.envRunner.debugger is not None:
+        if self.env.debugger is not None:
             self.env.skip() # valgrind is not working correctly with replication
 
     def test_CRUD_replication(self):
+        self.env.skipOnCluster()
```
Comment (AviAvni) [inline tests/flow/test_replication.py:21]: please recheck and add comment

File: tests/flow/test_replication_states.py [hunk 5]
```diff
@@ -92,6 +92,7 @@ def _choose_random(self, iter, k):
         return iter
 
     def test_replication_permutations(self):
+        self.env.skipOnCluster()
```
Comment (AviAvni) [inline tests/flow/test_replication_states.py:95]: use restoremodaux

File: tests/flow/test_results.py [hunk 6]
```diff
@@ -161,6 +161,8 @@ def test08_return_duplicate_expression(self):
 
     # Test implicit result-set size limit
     def test09_implicit_resultset_limit(self):
+        # CONFIG command on redis cluster is excluded
+        self.env.skipOnCluster()
```
Comment (AviAvni) [inline tests/flow/test_results.py:165]: need to check if which default config values can be passed to rlec cluster for redisgraph

File: tests/flow/test_ro_query.py [hunk 7]
```diff
@@ -12,7 +12,8 @@ def checkSlaveSynced(env, masterConn, slaveConn, graph_name):
 class test_read_only_query(FlowTestsBase):
     def __init__(self):
         self.env = Env(decodeResponses=True, useSlaves=True)
-        if self.env.envRunner.debugger is not None:
+        self.env.skipOnCluster()
```
Comment (AviAvni) [inline tests/flow/test_ro_query.py:15]: please add doc or re check

File: tests/flow/test_stress.py [hunk 8]
```diff
@@ -81,7 +81,9 @@ class testStressFlow():
     def __init__(self):
         self.env = Env(decodeResponses=True)
         # skip test if we're running under Valgrind
-        if self.env.envRunner.debugger is not None or os.getenv('COV') == '1':
+        # CONFIG command on redis cluster is excluded
+        self.env.skipOnCluster()
```
Comment (AviAvni) [inline tests/flow/test_stress.py:85]: check if we can understand the value of the config

**Review by AviAvni — CHANGES_REQUESTED** [review]
great work thanks see comments

File: .circleci/config.yml [hunk 9]
```diff
@@ -398,6 +402,66 @@ jobs:
...
+            du -ah --apparent-size bin/artifacts/*
+      - run:
+          name: Start RLEC
+          shell: /bin/bash -l -eo pipefail
+          command: |
+            cd
+            mkdir -p rlec && chmod 777 rlec
+            ./RedisGraph/deps/readies/bin/xtx -dMODULE=$(cd RedisGraph/bin/artifacts && ls *.zip) \
+                RedisGraph/tests/flow/rlec/redis-modules.yml > rlec/redis-modules.yml
+            rlec start -s 1 -n 1 --verbose --version 6.2.8 --build 53 --os bionic
```
Comment (rafie) [inline .circleci/config.yml:454]: We can use newer version and drop the build number:
```
rlec start -s 1 -n 1 --verbose --version 6.2.12 --os bionic
```

**Pull request closed without merging**